api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -65,14 +69,36 @@ def get_or_create_organisation_hubspot_id(self, organisation: Organisation) -> s
         if getattr(organisation, "hubspot_organisation", None):
             return organisation.hubspot_organisation.hubspot_id
 
-        response = self.client.create_company(name=organisation.name)
+        response = self.client.create_company(
+            name=organisation.name,
+            active_subscription=organisation.subscription.plan,
+        )
+
         # Store the organisation data in the database since we are
         # unable to look them up via a unique identifier.
         HubspotOrganisation.objects.create(
             organisation=organisation,
             hubspot_id=response["id"],
         )
+
         return response["id"]
 
+    def update_company_active_subscription(self, subscription: Subscription) -> dict:
+        if not subscription.plan:
+            return
```

Comment:
This return contradicts the type definition. Not sure if the solution would be to raise an exception here instead or just to change the return type to `dict | None`

Reply:
Nice catch. And yeah I think updating the return type is the right call. An exception would just be an annoyance for the caller.